parts/linux/cloud-init/artifacts/components.json

```diff
@@ -590,7 +590,7 @@
             "versionsV2": [
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.1.9-5.cm2"
+                "latestVersion": "1.1.9-7.cm2"
```

Comment:
we're gonna remove runc entries for mariner/azlinux right?

Reply:
I will remove it in my PR